### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EladKohavi**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=CrH0H_q)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,92 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
...
+##Elad's history:
+I was born in 1990
+##Elad's history:
+I was born in 1990
+##Elad's history:
+I was born in 1990
+##Elad's history:
+I was born in 1990
+##Elad's history:
+I was born in 1990
```
Comment (yeelali14) [inline README.md:102]: lol

File: README.md [hunk 2]
```diff
@@ -14,6 +14,92 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
...
```
Comment (yeelali14) [inline README.md:88]: lollll

**Review by yeelali14 — CHANGES_REQUESTED** [review]
LGTM

**Comment by yeelali14** [comment]
comment

File: README.md [hunk 3]
```diff
@@ -14,6 +14,92 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 ##Elad's history:
 I was born in 1990
 
+##Elad's history:
+I was born in 1990
+##Elad's history:
+I was born in 1990
+##Elad's history:
+I was born in 1990
```
Comment (yeelali14) [inline README.md:22]: WTF

**Review by yeelali14 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (EladKohavi) [reply README.md:22]: FTW

File: README.md (hunk 2 above)
Comment (EladKohavi) [reply README.md:88]: hghddfs

**Comment by EladKohavi** [comment]
comet or comment?

File: README.md (hunk 3 above)
Comment (yeelali14) [reply README.md:22]: lol

**Review by yeelali14 — APPROVED** [review]
bug

**Pull request closed without merging**